docs/source/data_import_export.rst

```diff
@@ -113,12 +113,23 @@ Be careful not to press this button unless you want to wipe the whole specificat
 When a table is selected, it's data and a preview of how the selected mapping will
 import the data will be presented under the options dock widget. An important aspect of data import is
 whether each item in the input data should be read as a string, a number,
-a time stamp or something else. By default all input data is read as strings.
+a time stamp or something else. In other words, the importer should know the data types of the input data.
+By default all input data is read as strings.
 However, more often than not things like parameter values are actually numbers. Though types are usually casted automatically,
 it is possible to manually control what type of data each column (and sometimes each row) contains from the preview table.
 Clicking the data type indicator button on column or row headers pops up a menu with a selection of available data types.
 Right clicking the column/row header also gives the opportunity to change the data type of all columns/rows at once.
 
+The input data should conform to the specified data type. For float (number) type the decimal separator 
+should be point. For datetimes such as time stamps the recommended format is ISO8601 (e.g. 2020-03-01T01:00). 
+If non-ISO8601 format is 
+detected, the importer falls back to the dateutil Python library which supports a lot of different formats.
+However, ambiguous formats can be interpreted wrongly. For example, 01-03-2020 is interpreted as January 3, 2020.
+If the source file is an Excel file, date cells are interpreted correctly although the way the are shown in
+Excel can be ambiguous. For time durations you can use units "s", "m", "h" and "D", e.g. "4h". 
```

Comment:
For full specification of what is an acceptable duration, see [here](https://spine-database-api.readthedocs.io/en/latest/parameter_value_format.html#duration), so Importer should also accept durations such as `14 days`. The last point mentioning that plain integers are converted to minutes does not seem to work, though. There is also [an issue](https://github.com/spine-tools/Spine-Database-API/issues/321) about dropping year and month units so I am unsure if we want to encourage users to use those units.

In any case, maybe you could mention here that a 'long' unit is acceptable (also its plural form), i.e. "second", "minute", "hour" and "day".